### PR TITLE
port output scripts to dev folder

### LINKED ISSUE #6031: Port ember-new-output scripts to /dev/
Tracker for me to get this done. The source is https://gist.github.com/nathanhammond/e0a55b4d0328b45b8ef5. It may need some massaging for our release channels.

### PR DESCRIPTION
**Pull request by kellyselden**
Resolves #6031

### CODE REVIEW

File: dev/add-to-output-repos.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+EMBERVERSION=`ember version | grep "ember-cli:" | cut -d' ' -f2`
+
+commands=(new addon)
+
+for i in ${commands[@]}; do
+  command=$i
+  repo_folder=ember-$command-output
+
+  local_folder=my-$command
```
Comment (nathanhammond) [inline dev/add-to-output-repos.sh]: Can we make this `/tmp/my-$command` as it's [safely out of the scope](https://github.com/ember-cli/ember-cli/blob/master/.gitignore#L3) of accidental self-trolling via `git`?

**Comment by nathanhammond** [comment]
One comment. Other than that, LGTM, self-ship when addressed!

**Comment by kellyselden** [comment]
I don't know why this is targeting master...

**Comment by nathanhammond** [comment]
Are you going to reopen a new version of this?

**Pull request closed without merging**